libs/novui/src/components/toggle/Toggle.tsx

```diff
@@ -0,0 +1,34 @@
+import { PolymorphicRef } from '../../types/props-helpers';
+import { ChangeEvent, forwardRef, ReactNode } from 'react';
+import { Switch } from '@mantine/core';
+import { css, cx } from '../../../styled-system/css';
+import { toggle } from '../../../styled-system/recipes';
+import { splitCssProps } from '../../../styled-system/jsx';
+
+interface IToggleProps {
+  label?: ReactNode;
+  checked?: boolean;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  className?: string;
+  disabled?: boolean;
+}
+
+export const Toggle = forwardRef((props: IToggleProps, ref?: PolymorphicRef<'input'>) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** How do you about calling it Switch? To make it easier to follow what we already have‏

Reply:
When panda complies this down it uses `switch` as variable name and typescript complains about it being a reserved word, so I like to stick with toggle here :)